### ISSUE TITLE
Add "coming up next"/"on deck" display to overlay

### ISSUE THREAD

**Issue by dguenther** [dguenther/web-overlay#44]
It'd be nice to be able to enter two players for a "coming up next" display. The "on deck" display could also have a title, like "winner's finals". I think the display would be on its own page, and you'd activate it with scene select in OBS.

Could also add another select box to the challonge section to select the on deck match.

**Comment by Camtendo** [comment]
This is an awesome idea and I'll probably get to it sooner than later. The only confusing part would be trying to put a match on deck where one of the participants is unknown.

**Comment by dguenther** [comment]
Yeah. A good starting point would be non-challonge stuff, then we could figure out what we want to do with challonge. Maybe try to populate the on-deck box once challonge figures out who the other participant will be?
